app/controllers/posts_controller.rb

```diff
@@ -1,19 +1,20 @@
 class PostsController < ApplicationController
   def index
+    # @user = User.includes(:posts).find(params[:user_id])
```

Comment:
- Please remove comments from your code. They make code look ugly.
- Add `load_and_authorize_resource` as a `before_action` so `cancancan` is avaliable to your controller